src/modules/RA/Viewpoint/components/ViewpointFields.js

```diff
@@ -73,7 +80,11 @@ const ViewpointFields = ({ edit, classes, mapConfig, ...props }) => {
 
         <TextInput source="properties.voie" />
 
-        {waiting && <><LinearProgress /></>}
+        {waiting && (
+          <>
+            <LinearProgress />
+          </>
```

Comment:
Is this fragment really required?

Reply:
Yes, to protect LinearProgress from ReactAdmin props automatically given to all childs